components/applications-service/cmd/applications-publisher/main.go

```diff
@@ -102,6 +102,10 @@ func main() {
 	flag.Usage = usage
 	flag.Parse()
 
+	if uniqID {
```

Comment:
We were assigning this variable too late for it to affect anything. We want to assign before it's used on line 112 (updated code)